### ISSUE TITLE
How to improve multilingual ocr outcome?

### ISSUE THREAD

**Issue by Kate-D-S** [tesseract-ocr/tesseract#3919]
I need to OCR some poor quality documents which contain different alphabets e.g. german/polish/english.
I did OCR with all of the alphabets at first because I thought it would be faster (I mean: -l deu+pol+eng ).

I noticed that some characters were misidentified so I thought that result will be better after reducing the number of alphabets to those that de facto appear in the document (I checked it manually). 
But after the reduction of languages, there are characters replaced incorrectly with other characters even though the first identification was correct.

### Current Behavior:
-l eng+pol+deu
the result: `DÖNĘR` 

-l eng+deu
the result: `DONER`

### Expected Behavior:
The correct spelling: `DÖNER`

Is there anything what I possibly could do to improve the outcome? any additional parameters which I could use? 

------------------------

### Environment

* **Tesseract Version**: Tesseract Version: 4.1.1

**Comment by stweil** [comment]
I suggest to use the supported latest version 5.2 with a model which supports all Latin based languages, for example `script/Latin` (or `script/Fraktur` for historic texts).

**Comment by Kate-D-S** [comment]
Why do you think using scripts it's better then single language? How is that work? Those are models for whole alphabets?

**Comment by stweil** [comment]
Yes, `script/Latin` is basically a combination of all languages which use Latin script (English, Spanish, Italian, French, German and more).

**Comment by Kate-D-S** [comment]
Is there any list where I can check what languages are covered by which script?

**Comment by amitdo** [comment]
See here:  https://github.com/tesseract-ocr/tessdata/issues/62#issuecomment-319442674

AFAIK, there is no full detailed list.

You can extract any traineddata file by using:

`combine_tessdata -e ...`

Then you can examine the unicharset file and the dawg file.

**Comment by Kate-D-S** [comment]
Isn't that answer on my own question then? 

https://github.com/tesseract-ocr/langdata_lstm/blob/main/script/Latin.langs.txt

**Comment by Kate-D-S** [comment]
Do you possible know why script could be better then combination of single languages? (I mean, why script/Latin could be better then -l eng+deu+pol ? ) I mean some technical explanation :) how tesseract works?

**Comment by amitdo** [comment]
>Isn't that answer on my own question then? 

>https://github.com/tesseract-ocr/langdata_lstm/blob/main/script/Latin.langs.txt

I was not aware of this file :)

The other scripts does not have a `langs.txt` file.

**Comment by Kate-D-S** [comment]
All scripts have it :) 
https://github.com/tesseract-ocr/langdata_lstm/tree/main/script 
(at the bottom of the page)

**Comment by amitdo** [comment]
:)

I didn't notice it. I entered into a few folders and didn't see langs.txt like the Latin folder has.
